servers/src/main/java/tachyon/master/journal/JournalTailerThread.java

```diff
@@ -79,8 +79,17 @@ public void run() {
         long waitForShutdownStart = -1;
 
         // Load the checkpoint file.
+        LOG.info("Waiting to load the checkpoint file.");
         JournalTailer tailer = new JournalTailer(mMaster, mJournal);
+        while (!tailer.checkpointExists()) {
```

Comment:
Otherwise, `IOException` will be thrown in `processJournalCheckpoint`, the exception will be caught and error message will be logged, then there will be too many unnecessary messages telling you the checkpoint file can not be found...